static/js/dashboard_routes.js

```diff
@@ -0,0 +1,99 @@
+import App from './containers/App';
+
+const errorLoading = error => {
+  throw new Error(`Dynamic page loading failed: ${error}`);
+};
+
+const loadRoute = cb => (
+  module => cb(null, module.default)
+);
+
+const username = SETTINGS.user ? SETTINGS.user.username : '';
+
+export const routes = {
+  path: "/",
+  component: App,
+  childRoutes: [
+    {
+      path: 'dashboard',
+      getComponent(location, cb) {
```

Comment:
Can you rename `location` to `nextState` to match name in the documentation [here](https://github.com/ReactTraining/react-router/blob/master/docs/API.md#getcomponentnextstate-callback)?